server/utils/vectorDbProviders/chroma/index.js

```diff
@@ -290,7 +292,8 @@ const Chroma = {
       client,
       namespace,
       queryVector,
-      similarityThreshold
+      similarityThreshold,
+      topN || 4
```

Comment:
If we are going to fallback to 4 anyway - just have that be the fallback in the function arguments